src/services/delivertTest/deliveryTest.ts

```diff
@@ -7,7 +7,7 @@ import stats from '../../util/stats';
 import logger from '../../logger';
 import tags from '../../v0/util/tags';
 
-export default class DeliveryTestService {
+export class DeliveryTestService {
   public static async doTestDelivery(
```

Comment:
how are the test cases passing as `version` should also be passed now ?
https://github.com/rudderlabs/rudder-transformer/pull/2865/files#diff-223989b90529aa3a55fbd07c48aa218b7ece2e1d77fadb70805dbf28f0cd31d5R18

Should add some assertions in proxy destination test cases on whether genericNetworkHandler is expected or not ?

Reply:
Endpoint defaulting to `v0` is fine but the comment is about `getNetworkHandler` call from `deliveryTest.ts`
As version is not being passed, it returns `handlers.generic`